pipeval/validate/validate.py

```diff
@@ -24,7 +25,7 @@
     'file-cram': ['.cram'],
     'file-vcf': ['.vcf', '.vcf.gz'],
     'file-fasta': ['.fasta', '.fa'],
-    'file-fastq':['.fastq', '.fq.gz', '.fq', '.fastq.gz'],
+    'file-fastq':['.fastq', '.fq.gz', '.fq', '.fastq.gz', '.fastq.bz2', '.fq.bz2'],
```

Comment:
Do aligners generally accept bzip2-compressed FASTQ files? Also, we might want to throw a warning when FASTQs are not compressed, or even error out by default, to raise awareness of storage usage..

Reply:
Generally, I don't believe aligners directly accept bzip2-compressed files but they seem to support decompressing on-the-fly within the command so I included the option to validate FASTQ files that may be compressed with bzip2. Even if aligners don't directly accept the compression, external users may still have FASTQ files in that format so I included the option to validate those formats directly.

On the point of erroring on uncompressed FASTQ files, I agree with Aaron that that's generally not something PipeVal natively should be concerned with as an open-source validation tool. Internally, it makes sense to develop an add-on for that behavior though

Reply:
Right, this function is to validate the file format so adding a plug-in or something would make more sense.